### PR TITLE
Adopt node-red 1.0=< schema

### PR DESCRIPTION
**Pull request by zinen**
Schema can be found here https://nodered.org/docs/creating-nodes/node-js

- Adopt node-red 1.0=< schema bff7b94ab91ca1234ae3cc1d6aa6d733b776e39c
- Dont send anything to console that can be catched and send to node-red console 29138bd3de133e9b0279029532baccf2ec43b59e
- Cleaned up unused code ec19c5dfda2bf7b83ed41d739b4fd1a34dd2d1ea

### CODE REVIEW

File: postgresql.js [hunk 1]
```diff
@@ -247,10 +253,17 @@ module.exports = function (RED) {
 
 								handleDone();
 								downstreamReady = false;
-								node.send(msg);
 								if (tickUpstreamNode) {
 									tickUpstreamNode.receive({ tick: true });
 								}
+								if (done) {
```
Comment (Alkarex) [inline postgresql.js:259]: This section should be moved up to where `node.send(msg);` was, before allowing the reception of a new message (tick).

**Review by Alkarex — CHANGES_REQUESTED** [review]
Small comment about code location

**Comment by Alkarex** [comment]
Thanks @zinen , looks good 👍🏻

File: postgresql.js (hunk 1 above)
Comment (zinen) [reply postgresql.js:260]: Fixed!
I did have read up on then `done` call here https://nodered.org/blog/2019/09/20/node-done to make sure I did it right. To my understanding the `done` call should be at the very end of the node

**Comment by Alkarex** [comment]
Ok :-) There are two remaining `node.send(...);` instances, which should probably be replaced by the approach proposed on
https://nodered.org/blog/2019/09/20/node-done#backwards-compatibility

**Comment by zinen** [comment]
`node.send(...);` now changed af46cee. But that ment a duplicated variable/function had to be renamed 50b535a.

**Comment by zinen** [comment]
Done should be called along with complete part. Wait for it I'll fix

**Comment by zinen** [comment]
Done :)

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
Thanks. I will test a bit and merge.

**Comment by Alkarex** [comment]
@zinen I have made some more changes, including support for named parameters https://github.com/alexandrainst/node-red-contrib-postgresql/pull/16
I will soon make an official release pushed to npm, so if you have any chance of testing the current code on your side prior to that, it would be appreciated .-)

**Comment by zinen** [comment]
@Alkarex tested a bit and version 0.6.0 (since commit b55c5ed9f0ef759ff0c9383cd8e5a54e607c0cbf) looks good to me. 
- Tested error catching as of node>1.0.
- Tested "complete" catching as of node>1.0.
- And some other random queries primary with external `msg.query` paramenter.

All looks good.

**Comment by Alkarex** [comment]
@zinen Thanks for the feedback :-)

**Pull request merged**